README.md

```diff
@@ -66,13 +66,13 @@ will see 3G device memory inside container
 
 ### Supported devices
 
-[![nvidia GPU](https://img.shields.io/badge/Nvidia-GPU-blue)](https://github.com/Project-HAMi/HAMi#preparing-your-gpu-nodes)
-[![cambricon MLU](https://img.shields.io/badge/Cambricon-Mlu-blue)](docs/cambricon-mlu-support.md)
-[![hygon DCU](https://img.shields.io/badge/Hygon-DCU-blue)](docs/hygon-dcu-support.md)
-[![iluvatar GPU](https://img.shields.io/badge/Iluvatar-GPU-blue)](docs/iluvatar-gpu-support.md)
-[![mthreads GPU](https://img.shields.io/badge/Mthreads-GPU-blue)](docs/mthreads-support.md)
-[![ascend NPU](https://img.shields.io/badge/Ascend-NPU-blue)](https://github.com/Project-HAMi/ascend-device-plugin/blob/main/README.md)
-[![metax GPU](https://img.shields.io/badge/metax-GPU-blue)](docs/metax-support.md)
+[nvidia GPU](https://github.com/Project-HAMi/HAMi#preparing-your-gpu-nodes)   
```

Comment:
According to the [guidelines](https://www.nvidia.com/content/dam/en-zz/Solutions/about-us/documents/NVIDIA-Trademark-and-Logo-Usage-Guidelines.pdf) from NVIDIA, nvidia here should be NVIDIA.

Please check if the name of other devices should also be updated.